### ISSUE TITLE
iOS - PDF Reader loads the first pdf read

### ISSUE THREAD

**Issue by tiagoAGA** [sitewaerts/cordova-plugin-document-viewer#43]
I have a collection of PDF's URL's.

After open the first PDF, when I open other PDF the plugin seems to hold the first one in cache, rendering the pages of first PDF and only after render the correct pages.

It happened with anyone?

Thank you all.

**Comment by regnete** [comment]
I dont' exactly understand what kind of problem you described. Please check if your issue is duplicate to #37, #33, #13.
 These issues are solved in version 0.8.0. 

Which version of the plugin are you using?
Wich OS?
Ca you reproduce the bg with the sample app https://github.com/sitewaerts/cordova-plugin-document-viewer-sample-app?

**Comment by tiagoAGA** [comment]
Hi regnete.
Thank you for your reply.
Yes, it's the same problem reported on #13 case B).

there's no way to solve the cache issues when the file name and file url are the same?

Thanks for your help.

**Comment by regnete** [comment]
Currently there is no way to flush the cache.
Worarround: create a random file name before starting the plugin.

**Comment by tiagoAGA** [comment]
My solution is remove the old file, and use different names. Otherwise I would overload the file url.

Other issue: Do you know why sometimes the page list on the bottom loads blank pages instead the pdf page? 

Thank you for your support.

**Comment by regnete** [comment]
please create a new issue for the "blank pages" effekt. anyway we need a reproducable bug decription, e.g inlcluding sample documents.
